metaflow/tutorials/09-hellokfp/spark_flow.py

```diff
@@ -0,0 +1,44 @@
+from typing import List, Tuple
+from metaflow import FlowSpec, step, resources, spark, Parameter
+from metaflow.plugins.kfp.kfp import SparkSessionCreator
+
+class SparkFlow(FlowSpec):
+    """
+    A flow which uses the @spark decorator to run a spark job within a single step.
+    """
+    spark_eventLog_enabled = Parameter(
+        'spark_eventLog_enabled',
+        help='param with default',
+        default="false",
+    )
```

Comment:
Can `Parameters` be complex objects (ie. `dict` or `list`) or are they limited to just simple types (ie. `int`, `float`, `bool`, `str`, etc)?

Reply:
They can be `dict` and `list` as well.